articles/systems/08-platform-framework.md

```diff
@@ -0,0 +1,49 @@
+# Mixed Reality Toolkit Platform Framework
+
+A major component of the Mixed Reality Toolkit is the Platform Framework that was employed to both accurately determine what platforms have been added to the scope of the project and also to determine when a platform is active/running.
+
+## Platform Selection
+
+![](../images/Configuation/Platforms/PlatformSelector.png)
+
+Wherever configuration can have a variant for platforms, to make the configuration only active when selected platforms are available, the Platform selector is employed.
+
+The list support multiple selections (for cases where a system or profile) can operate for multiple platforms.
+
+The list is dynamic and will update based on which XRTK platforms have been installed and what platforms are supported by the developers Unity installation.
+
+## Platform Identification
+
+To explain how this system basically operates, there is a `Platform` script in the UPM package runtime folder which implements the required interfaces for the platform system, including:
+
+```csharp
+    public override bool IsAvailable;
```

Comment:
why duplicate what's already written in code?

if this was to change in the future it'd be high upkeep

Reply:
As it's new and we are introducing it, it is worthwhile to highlight it. We can always update or remove this section later as we evolve.